setup.py

```diff
@@ -42,7 +42,7 @@
     "retrying>=1.3.4,<2",
     "scikit-learn>=1.2.0,<2",  # requires scipy, and introduce breaking change affects feature_extraction.text.TfidfVectorizer.min_df
     "scikit-surprise>=1.1.3",
-    "scipy>=1.10.1",
+    "scipy>=1.10.1,<=1.13.1",
```

Comment:
How will we remember that we need to change this when Cornac updates its version?

Reply:
I created the issue #2128 as a reminder, and added a comment in line 45.